src/Analyzers/Analyzers/src/UseParameterAttributeCorrectlyAnalyzer.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.AspNetCore.Analyzers.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.AspNetCore.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class UseParameterAttributeCorrectlyAnalyzer : DiagnosticAnalyzer
```

Comment:
Should this go in https://github.com/dotnet/aspnetcore/tree/main/src/Components/Analyzers instead?